docs/source/markdown/podman-run.1.md

```diff
@@ -265,6 +265,12 @@ Memory nodes (MEMs) in which to allow execution. Only effective on NUMA systems.
 For example, if you have four memory nodes (0-3) on your system, use **--cpuset-mems=0,1**
 to only use memory from the first two memory nodes.
 
+#### **--dep**=**container**
+
+Specify one or more dependency containers.
+Dependency containers must be running before this container is started.
+The `podman run` command will automatically start these dependencies, but the `podman start` command must be instructed to do so by using the `--recursive` flag.
```

Comment:
I find this sentence to be confusing.  You're saying run automatically starts the dependencies, but the start command must be called with the recursive option first?  Won't that start command start the containers when it's called?

Would it make sense to add a `--recursive` option to run and just forgo the start command if you're doing a run (or perhaps call it under the covers?)